CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ if(BUILD_NVFUSER_BENCHMARK)
 endif()
 
 # set CUDA_ARCH for cu tests.
-if(EXISTS ${TORCH_CUDA_ARCH_LIST})
```

Comment:
Without this change, the passed target list of cuda archs was incorrectly handled, as if it was a path to files.